test_files/preprocessor.h

```diff
@@ -0,0 +1,4 @@
+
+#define FOO 0
+#define FOOBAR 1
+#define FOOBARBAZ 2
```

Comment:
So perhaps the implementation should check the token kind and only translate the ones that can be translated to D.